def sum(x,y):
    #~ z = x+y
    raise Exception("Test msg")
    
sum(1,"b")
#~ try:
    #~ sum(1,"b")
#~ except Exception as err:
    #~ print err
    